opm/input/eclipse/EclipseState/Grid/FieldProps.cpp

```diff
@@ -691,144 +775,185 @@ void FieldProps::distribute_toplayer(Fieldprops::FieldData<double>& field_data,
         return true;
+    }
 
-    if (Fieldprops::keywords::PROPS::satfunc.count(keyword) != 0)
+    if (Fieldprops::keywords::PROPS::satfunc.count(keyword) != 0) {
         return true;
+    }
 
-    if (Fieldprops::keywords::SOLUTION::double_keywords.count(keyword) != 0)
+    if (Fieldprops::keywords::SOLUTION::double_keywords.count(keyword) != 0) {
         return true;
+    }
 
     return false;
 }
 
 template <>
-bool FieldProps::supported<int>(const std::string& keyword) {
-    if (Fieldprops::keywords::REGIONS::int_keywords.count(keyword) != 0)
+bool FieldProps::supported<int>(const std::string& keyword)
+{
+    if (Fieldprops::keywords::REGIONS::int_keywords.count(keyword) != 0) {
         return true;
+    }
 
-    if (Fieldprops::keywords::GRID::int_keywords.count(keyword) != 0)
+    if (Fieldprops::keywords::GRID::int_keywords.count(keyword) != 0) {
         return true;
+    }
 
-    if (Fieldprops::keywords::SCHEDULE::int_keywords.count(keyword) != 0)
+    if (Fieldprops::keywords::SCHEDULE::int_keywords.count(keyword) != 0) {
         return true;
+    }
 
     return Fieldprops::keywords::isFipxxx(keyword);
 }
 
 template <>
-Fieldprops::FieldData<double>& FieldProps::init_get(const std::string& keyword_name, const Fieldprops::keywords::keyword_info<double>& kw_info,
-                                                    bool multiplier_in_edit) {
-    
-    if(multiplier_in_edit && !kw_info.scalar_init.has_value())
-        OPM_THROW(std::logic_error, std::string("Keyword " +  keyword_name + " is a multiplier and should have a default initial value."));
-    const std::string& keyword = Fieldprops::keywords::get_keyword_from_alias(keyword_name);
-    const std::string& mult_keyword = std::string(multiplier_in_edit? getMultiplierPrefix() : "") + keyword;
+Fieldprops::FieldData<double>&
+FieldProps::init_get(const std::string& keyword_name,
+                     const Fieldprops::keywords::keyword_info<double>& kw_info,
+                     const bool multiplier_in_edit)
+{
+    if (multiplier_in_edit && !kw_info.scalar_init.has_value()) {
+        OPM_THROW(std::logic_error, "Keyword " +  keyword_name +
+                  " is a multiplier and should have a default initial value.");
+    }
+
+    const auto keyword = Fieldprops::keywords::get_keyword_from_alias(keyword_name);
+    const auto mult_keyword = std::string(multiplier_in_edit ? getMultiplierPrefix() : "") + keyword;
 
     auto iter = this->double_data.find(mult_keyword);
     if (iter != this->double_data.end()) {
         return iter->second;
-    } else if(multiplier_in_edit){
-        assert(keyword != ParserKeywords::PORV::keywordName && keyword != ParserKeywords::TEMPI::keywordName &&
-               (Fieldprops::keywords::PROPS::satfunc.count(keyword) != 1) && !is_capillary_pressure(keyword));
-        multiplier_kw_infos_[mult_keyword] = kw_info;
+    }
+    else if (multiplier_in_edit) {
+        assert((keyword != ParserKeywords::PORV::keywordName) &&
```

Comment:
general niggle. asserting with && makes assertion less useful, multiple statements is better..